app/views/admin/roasts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
+
+<h1>List of All Roasts</h1>
+
+<%= link_to new_admin_roast_path, {:class => "btn btn btn-block"} do %>
+  <span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span> Add New Roast
+<% end %>
+
+<table class="table table-hover">
+  <thead>
+    <tr>
+      <th>Company</th>
+      <th>Roast Name</th>
+      <th>Description</th>
+      <th colspan="2"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @roasts.each do |roast| %>
+      <tr>
+        <td><%= roast.company %></td>
+        <td><%= roast.roastName %></td>
```

Comment:
To follow normal rails convention this should be called roast_name or name instead of roastName (snake_case)